ports/windows/Makefile

```diff
@@ -84,6 +84,11 @@ ifneq ($(FROZEN_MANIFEST),)
 CFLAGS += -DMICROPY_QSTR_EXTRA_POOL=mp_qstr_frozen_const_pool -DMICROPY_MODULE_FROZEN_MPY=1 -DMPZ_DIG_SIZE=16
 endif
 
+ifeq ($(shell $(CC) -dumpmachine),i686-w64-mingw32)
+# https://lemire.me/blog/2020/06/26/gcc-not-nearest
+CFLAGS += -msse -mfpmath=sse -march=pentium4
```

Comment:
This needs a more substantial comment, eg "force gcc to use IEEE correct rounding when optimising float constants at compile time"

Reply:
updated